### DISCUSSION TITLE
can laragon pretty url pointing to another folder outside of document root?

### DISCUSSION THREAD

**Discussion by uluumbch** [leokhoa/laragon#849] [Q&A]
The default document root for laragon is in "LARAGON_INSATALLED/www". I know when creating a new project inside this folder, laragon will create me virtualhost and add host to host file in windows.
but is there any script or things that can make project inside laragon document root folder can also use pretty url?
for example, I want all my project in "D:\projects\" will be created a pretty urls too.

I tried manually create virtualhost to "LARAGON_INSTALLED\etc\apache2\sites-enabled" but it still cannot use the pretty url

**Comment by ppfoong** [comment]
Try change Laragon's preference > Document Root.
